### PR TITLE
Replaced the js alert with bootstrap modal

### PR DESCRIPTION
**Pull request by amisha296**
Targets #1784

### Description
I have replaced the Js alert box with bootstrap modal and made an additional function to set the index of the user, which would trigger when we click on remove user, and the modal will open.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by P4NK4J — COMMENTED** [review]
@amisha296 looks good. Left a minor comment.

File: Modules/User/Resources/assets/js/components/UserListing.vue [hunk 1]
```diff
@@ -96,17 +110,23 @@ export default {
...
 
-		removeUser: async function(index) {
+		removeUser: async function() {
 
-			if(!confirm("Are you sure?")) {
-				return true;
-			}
+			// if(!confirm("Are you sure?")) {
+			// 	return true;
+			// }
```
[the remark this answers is not in the document]
Comment (amisha296) [reply Modules/User/Resources/assets/js/components/UserListing.vue:117]: okay

[the remark this answers is not in the document]
Comment (P4NK4J) [reply Modules/User/Resources/assets/js/components/UserListing.vue:117]: @amisha296 I can still see the commented code here, could you please delete it ?

**Review by P4NK4J — APPROVED** [review]
looks good.

**Pull request merged**